src/routes/api/trips.routes.js

```diff
@@ -78,37 +78,44 @@ router.post('/request', [checkUser, dateValidator, tripRequestValidator], tripsC
  *   post:
  *     security: []
  *     summary: Multi city trip request
- *     description: trip requests
+ *     description: Create a multi city trip request
  *     tags:
  *       - Trips
  *     requestBody:
  *       content:
  *         application/json:
  *           schema:
- *             type: array
- *             items:
- *               type: object
- *               properties:
- *                 type:
- *                   type: string
- *                 passportNumber:
- *                   type: string
- *                 gender:
- *                   type: string
- *                 fromCountry:
- *                   type: string
- *                 fromCity:
- *                   type: string
- *                 toCountry:
- *                   type: string
- *                 toCity:
- *                   type: string
- *                 accommodation:
- *                   type: string
- *                 departureDate:
- *                   type: string
- *                 reason:
- *                   type: string
+ *             type: object
+ *             properties:
+ *               type:
+ *                 type: string
+ *               fromCountry:
+ *                 type: string
```

Comment:
```suggestion
 *                 type: string
                    example: Rwanda
```
It's better to show examples for every property while documenting. It'll simplify the work of someone who will present on the demo

Reply:
@hakoemmy I have validated every input.

Reply:
@AbdoulNuru I understand, it was my suggestion. But still, it's OK